Worktez/worktez
# Fix/icon colour change

### Functionality:
Changed the color of Priority and Difficulty icons

### Solution:
Changed the icon colors of by changing the color code

### Risk level:

### How to test:
Add task for each Priority and difficulty and check whether the correct colors are displayed
